package/kernel/linux/modules/crypto.mk

```diff
@@ -809,3 +809,18 @@ endef
 
 $(eval $(call KernelPackage,crypto-xts))
 
+
+define KernelPackage/crypto-zstd
+  TITLE:=zstd compression CryptoAPI module
+  DEPENDS:=+kmod-lib-zstd +kmod-crypto-acompress
+  KCONFIG:=CONFIG_CRYPTO_ZSTD
+  FILES:=$(LINUX_DIR)/crypto/zstd.ko
+  AUTOLOAD:=$(call AutoLoad,09,zstd)
+  $(call AddDepends/crypto)
```

Comment:
I'm gonna test it soon.

Reply:
About this: modules do work with `AutoProbe`, but as long as the default OpenWrt packages use `AutoLoad`, it makes no sense at all. Mostly because the new kernel packages depend on the original packages and mixing the both options does not seems logical, unless someone can provide me a link to a official announcement of refactoring.